Scene/Scene2dInstance.cs

```diff
@@ -0,0 +1,44 @@
+﻿using RenderSharp.Math;
+using RenderSharp.RendererCommon;
+
+namespace RenderSharp.Scene
+{
+    internal class Scene2dInstance
```

Comment:
This should be public so other classes can access

Reply:
Do I remove "internal" and replace it with "public"?